app/controllers/provider_interface/offers_controller.rb

```diff
@@ -49,10 +49,10 @@ def show
     def update
       @wizard = OfferWizard.new(offer_store)
       if @wizard.valid?(:save)
-        ChangeAnOffer.new(actor: current_provider_user,
-                          application_choice: @application_choice,
-                          course_option: @wizard.course_option,
-                          offer_conditions: @wizard.conditions).save
+        ::ChangeOffer.new(actor: current_provider_user,
```

Comment:
should we move both those under the `app/services/provider_interface`?

Reply:
We decided to move away from the namespace, I think because a lot of the services are reused accross support and provider.